serveradmin/powerdns/models.py

```diff
@@ -20,3 +20,32 @@ class Domain(models.Model):
     class Meta:
         managed = False
         db_table = 'domains'
+
+
+class Record(models.Model):
+    """PowerDNS Record
+
+    Model to access records in the PowerDNS db.
+    """
+    TYPES = [
+        ('A', 'A'),
+        ('AAAA', 'AAAA'),
+        ('CNAME', 'CNAME'),
+        ('TXT', 'TXT'),
+        ('SSHFP', 'SSHFP'),
+        ('SOA', 'SOA'),
+        ('MX', 'MX'),
+        ('PTR', 'PTR'),
+        ('NS', 'NS'),
+    ]
+
+    id = models.BigIntegerField(primary_key=True)
+    domain_id = models.ForeignKey('Domain', on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, default=None)
+    type = models.CharField(max_length=10, default=None, choices=TYPES)
+    content = models.CharField(max_length=65535, default=None)
```

Comment:
If it is that big, might it be better to directly use a `text` field?

Reply:
Given by the PowerDNS schema (mind the model is not managed).